code/style.css

```diff
@@ -1,7 +1,212 @@
+  .img-tablet, .img-phone {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) { 
+  video, .img-phone {
+    display: none;
+  }
+}
+
+@media (max-width: 767px) { 
+  video, .img-tablet {
+    display: none;
+  }
+}
+
+h1 {
+  width: 100%;
+  position: absolute;
+  text-align: center;
+  color: white;
+  font-size: 22.1vw;
+
+  animation-name: lightning;
+  animation-duration: 0.1s;
+  animation-delay: 5s;
+  animation-iteration-count: 20;
+}
+
+@keyframes lightning {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  h1 {
+    animation-name: slowyellow;
+    animation-duration: 10s;
+    animation-delay: 2s;
+    animation-direction: alternate;
+    animation-iteration-count: infinite; 
+  }
+}
+
+@keyframes slowyellow {
+  from {color: white;}
+  to {color: yellow;}
+} 
+
+@media (max-width: 767px) {
+  h1 {
+    color: yellow;
+    animation: none;
+  }
+}
+
+/*** CONTENT: SIGNUP FORM AND ACCORDION ***/
+
+.content-container {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  padding: 40px 20px 40px 20px;
+} 
+
+h2 {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 40px;
+  color: rgb(134, 125, 125);
+  text-align: center;
+}
+
+.content-item {
+  width: 40%;
+}
+
+@media (max-width: 1023px) {
+  form {
+    margin-bottom: 40px;
+  }
+  .accordion-container {
+    text-align: center;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .content-item {
+    width: 60%;
+  }
+}
+
+@media (max-width: 767px) {
+  .content-item {
+    width: 80%;
+  }
+}
+
+/*** SIGNUP FORM ***/
+
+form {
+  height: 500px;
+  padding: 20px 40px 20px 40px;
+  border-radius: 20px;
+  box-shadow: 0 4px 8px 4px rgba(0, 0, 0, 0.2);
+  font-family: 'Open Sans', sans-serif;
+}
+
+input[type=text]  {
+  width: 100%;
+  margin: 4px 0 8px 0;
+  padding: 8px;
+  border: none;
+  border-radius: 20px;
+  background-color: rgb(240, 238, 238);
+}
+
+input[type=text]:focus, input[type=e-mail]:focus {
+  outline: none;
+}
+
+p {
+  margin: 8px 0 4px 0;
+}
+
+.checkbox {
+  margin-top: 16px;
+}
+
+span {
+  font-weight: 700;
+}
+
+span:hover {
+  cursor: pointer;
+  text-decoration-line: underline;
+}
+
+.submit {
+  display: block;
+  margin: 40px auto;
+  padding: 10px;
+  border-width: 0;
+  border-radius: 20px;
+  background-color:rgb(134, 125, 125);
+  color: white;
+  font-weight: 700;  
+  font-size: 16px;
 }
+
+.submit:hover {
+  cursor: pointer;
+  background-color: black;
+}
+
+/*** FAQ ***/
+
+.accordion-container {
+  height: 500px;
+  padding: 20px 40px 20px 40px;
+  border-radius: 20px;
+  background-image: linear-gradient(to right,white, rgb(240, 238, 238));
+}
+
+.accordion-item {
+  margin-bottom: 16px;
+  font-family: 'Architects Daughter', cursive;
+  font-size: 20px;
+}
+
+.question {
+  padding-bottom: 8px;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.answer {
+  display: none;
+  padding-bottom: 8px;
+} 
```

Comment:
Would be nice to the eyes if these looked a little more different from each other

Reply:
I agree!! Will spend a little time here.